docs/prysm-usage/get-more-peers.md

```diff
@@ -0,0 +1,59 @@
+---
+id: get-more-peers
+title: Get More Peers
```

Comment:
Would change title to Improving Peer-to-Peer Connectivity or something along those lines
Sidebar: Improving P2P connectivity (or similar)

Reply:
done